Plugins/Persistence/Collect-ScheduledTaskDetails.ps1

```diff
@@ -0,0 +1,49 @@
+<#
+
+.SYNOPSIS
+    Plugin-Name: Collect-ScheduledTaskDetails.ps1
+    
+.Description
+    Collects scheduled task details from the remote host (run times, last task result, etc.). This plugin
+    does not collect the scheduled task files themselves
+
+    To collect the scheduled task configuration files use the Collect-ScheduledTasks
+    plugin.
+
+.EXAMPLE
+
+    Power-Response Execution
+
+    Set ComputerName Test-PC
```

Comment:
`Test-PC` referenced directly in favor of `<list of computers>`? Also no HuntName?

Reply:
collect plugin, not hunt.